operators/osdk-getting-started.adoc

```diff
@@ -0,0 +1,54 @@
+// This assembly is included in the following assemblies:
+//
+// * n/a
+
+[id='getting-started-osdk']
+= Getting started with the Operator SDK
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+:context: osdk-getting-started
+
+toc::[]
+
+{nbsp} +
+This guide walks through an example of building a simple Memcached Operator
+(`memcached-operator`) using tools and libraries provided by the Operator SDK,
+and shows how to manage the new Operator's lifecycle from installation through
+updating to a new version.
+
+As an administrator of a Kubernetes-based cluster, you can accomplish this using
+two centerpieces of the Operator Framework:
+
+Operator SDK:: Assists developers in bootstrapping and building an Operator
+based on their expertise without requiring knowledge of Kubernetes API
+complexities.
+
+Operator Lifecycle Manager (OLM):: Helps Operator users install, update, and
+generally manage the lifecycle of all Operators (and their associated services)
```

Comment:
I don't think operator should be capitalized when it's standalone. :thinking: 
I think you can also drop the ()s

Reply:
Checking on this.